AUTHORS

```diff
@@ -543,3 +544,72 @@ Dennis Meckel <meckel@datenschuppen.de>
 Harshil Meena <harshil.7535@gmail.com>
 Micky <mickydroch@gmail.com>
 arghdos <arghdos@gmail.com>
+Jainul Vaghasia <jainulvaghasia@D-10-157-53-226.dhcp4.washington.edu>
+Michele Zaffalon <michele.zaffalon@gmail.com>
+Martha Giannoudovardi <maapxa@gmail.com>
+devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+steph papanik <spapanik21@gmail.com>
+Isuru Fernando <isuru.11@cse.mrt.ac.lk>
+msdousti <msdousti@gmail.com>
+Arif Ahmed <arif.ahmed.5.10.1995@gmail.com>
+Abdullah Javed Nesar <abduljaved1994@gmail.com>
+Lakshya Agrawal <zeeshan.lakshya@gmail.com>
+shruti <shrutishrm512@gmail.com>
+ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
+rohitrango <rohit.rango@gmail.com>
+Hong Xu <hong@topbug.net>
+Ivan Petuhov <ivan@ostrovok.ru>
+Nicolas Guarin-Zapata <nicoguarin@gmail.com>
+Alsheh <alsheh@rpi.edu>
+Marcel Stimberg <marcel.stimberg@ens.fr>
+Marcel Stimberg <marcel.stimberg@inserm.fr>
+Alexey Pakhocmhik <cool.Bakov@yandex.ru>
+Tommy Olofsson <tommy.olofsson.90@gmail.com>
+Zulfikar <zulfikar97@gmail.com>
+bsdz <bsdz@users.noreply.github.com>
+Danny Hermes <daniel.j.hermes@gmail.com>
+psqq <pestov-sa@yandex.ru>
+Amit Kumar <aktech@users.noreply.github.com>
+Mohit Chandra <mohit.chandra@research.iiit.ac.in>
+Gaurav Dhingra <gauravdhingra.gxyd@gmail.com>
+karthikchintapalli <karthik.chintapalli@gmail.com>
+Marcin Briański <marcin.brianski@student.uj.edu.pl>
+andreo <andrey.torba@gmail.com>
+Flamy Owl <flamyowl@protonmail.ch>
+Yicong Guo <guoyicong100@gmail.com>
+mamakancha <varun.garg03@gmail.com>
+madan96 <rishabhmadan96@gmail.com>
+Aditya Kapoor <aditya.kapoor.apm12@itbhu.ac.in>
+Karan Sharma <karan1276@gmail.com>
+Vedant Rathore <vedantr1998@gmail.com>
+Johan Blåbäck <johan.blaback@cea.fr>
+= <pranjaltale16@gmail.com>
+Pranjal Tale <pranjaltale16@gmail.com>
+jtoka <jason.tokayer@gmail.com>
+jtoka <jason.tokayer@capitalone.com>
+Raghav Jajodia <jajodia.raghav@gmail.com>
+Rajat Thakur <Rajat Thakur>
+rajatthakur <rajatthakur1997@gmail.com>
+Dhruv Bhanushali <dhruv_b@live.com>
+geety <anjul.ten@gmail.com>
+Markus <markus.mueller.1.g@googlemail.com>
+Rishabh Madan <rishabhmadan96@gmail.com>
+Barun Parruck <barun.parruck@gmail.com>
+Bao Chau <chauquocbao0907@gmail.com>
+tanayag <tanay_agrawal@hotmail.com>
+ranjithkumar007 <ranjith.dakshana2015@gmail.com>
```

Comment:
@smichr  , you can remove this redundant line [ranjithkumar007] here .. My full name is mentioned [below](https://github.com/mamakancha/sympy/blob/d0a9c81ee855ddfa46d6608f8b33db6d44cbe00f/AUTHORS#L614)